src/tests/sortingOnTheInventoryPage.spec.js

```diff
@@ -0,0 +1,46 @@
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+const { currentOrderOnPage, sortArray } = require('./utils');
+
+const sortingTypeParams = [
+    {
+    sortingTitle: 'A to Z',
+    sortingType: 'az',
+    },
+    {
+    sortingTitle: 'Z to A',
+    sortingType: 'za',
+    },
+    {
+    sortingTitle: 'Low to High',
+    sortingType: 'lohi',
+    },
+    {
+    sortingTitle: 'High to Low',
+    sortingType: 'hilo',
+    }
+    ]
```

Comment:
To make this more readable, you could update this array with the proper spacing.